mesh/mesh.hpp

```diff
@@ -574,16 +581,91 @@ class Mesh
    /// Move constructor, useful for using a Mesh as a function return value.
    Mesh(Mesh &&mesh);
 
-   /// Move assignment operstor.
+   /// Move assignment operator.
    Mesh& operator=(Mesh &&mesh);
 
    /// Explicitly delete the copy assignment operator.
    Mesh& operator=(const Mesh &mesh) = delete;
 
-   /** @name Named mesh constructors.
+   /// Construct a Mesh from the given primary data.
+   /** The array @a vertices is used as external data, i.e. the Mesh does not
+       copy the data and will not delete the pointer.
+
+       The data from the other arrays is copied into the internal Mesh data
+       structures.
+
+       This method calls the method FinalizeTopology(). The method Finalize()
```

Comment:
Does the user need to call anything after they change the vertices?

Reply:
No, but I can expand on this documentation to make this more clear.